loop_closure:
  max_icp_iterations: 100

map:
  # directory to save map in
  dir: "/home/patrick/maps/test_maps"

  comment: "slam6d_listening"

  # Truncation distance for the TSDF values (in m)
  max_distance: 0.6

  # Size of one grid cell in the discretised map (in m per cell)
  resolution: 64 #0.1 #0.03

  # localmap size for every dimension (in m)
  size:
    x: 20
    y: 20
    z: 15

  # Maximum weight for one TSDF cell. This variable ensures that later changes can still have an influence to the map
  max_weight: 10.0
